ui/src/components/Map/Map.jsx

```diff
@@ -28,66 +29,60 @@ const offsets = {
   DC: [49, 21]
 };
 
-function handleMouseClick(key) {
-  console.log(key);
-}
-
-
-
 const Map = () => {
   return (
     <ComposableMap projection="geoAlbersUsa">
       <Geographies geography={geoUrl}>
         {({ geographies }) => (
           <>
             {geographies.map(geo => (
-              <OverlayTrigger
-              trigger="click"
-              key={geo.rsmKey}
-              placement="right"
-              overlay={
-                <Popover id={`popover-positioned-right`}>
-                  <Popover.Header as="h3">{`Popover right`}</Popover.Header>
-                  <Popover.Body>
-                    <strong>Holy guacamole!</strong> Check this info.
-                  </Popover.Body>
-                </Popover>
-              }
-              >
-                <Geography
-                  key={geo.rsmKey}
-                  stroke="#FFF"
-                  geography={geo}
-                  fill="#DDD"
-                  onMouseDown={() => {
-                    handleMouseClick(geo.properties.name);
-                  }}
-                />
-              </OverlayTrigger>
+              <Geography
+                key={geo.rsmKey}
+                stroke="#FFF"
+                geography={geo}
+                fill="#DDD"
+                className="state"
+              />
             ))}
             {geographies.map(geo => {
               const centroid = geoCentroid(geo);
               const cur = allStates.find(s => s.val === geo.id);
               return (
+
+
                 <g key={geo.rsmKey + "-name"}>
                   {cur &&
                     centroid[0] > -160 &&
                     centroid[0] < -67 &&
                     (Object.keys(offsets).indexOf(cur.id) === -1 ? (
```

Comment:
use the 'in' operator
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in

Reply:
fixed this in next PR!